.github/workflows/build-wireguard-go-android.yml

```diff
@@ -1,20 +1,21 @@
 name: build-wireguard-go-android
-on: [workflow_dispatch, workflow_call]
+on:
+  workflow_dispatch:
+  workflow_call:
+  pull_request:
+    paths:
+      - ".github/workflows/build-wireguard-go-android.yml"
 
 env:
   CARGO_TERM_COLOR: always
 
 jobs:
   build:
-    runs-on: 'ubuntu-22.04'
+    runs-on: 'arc-ubuntu-22.04-dind'
```

Comment:
What is this runner?

Reply:
A new runner, basically same as arc-ubuntu-22.04 but with support for docker-in-docker